src/components/Footer.jsx

```diff
@@ -1,8 +1,52 @@
+import { useState } from "react";
 import { NavLink } from "react-router-dom";
 import logo from "/logo.png";
+import faqData from "../data/faqData.json";
+
 function Footer() {
+  const [openQuestion, setOpenQuestion] = useState(null);
+
+  const toggleQuestion = (index) => {
+    setOpenQuestion(openQuestion === index ? null : index);
+  };
+
   return (
     <>
+      {/* FAQ Section */}
+      <section className="bg-gradient-to-br from-gray-900 via-gray-800 to-gray-900 py-12">
+        <div className="mx-auto w-full max-w-3xl px-4">
+          <h2 className="text-2xl font-bold text-white mb-6 text-center">Frequently Asked Questions</h2>
+          <div className="space-y-3">
+            {faqData.map((faq, index) => (
+              <div key={index} className="bg-gray-800 bg-opacity-50 backdrop-blur-sm rounded-lg overflow-hidden border border-gray-700 transition-all duration-300 hover:border-gray-600">
+                <button
+                  className="flex justify-between items-center w-full text-left px-5 py-4"
+                  onClick={() => toggleQuestion(index)}
+                >
+                  <span className="font-medium text-white text-sm">{faq.question}</span>
+                  <svg
+                    className={`w-4 h-4 text-gray-400 transform transition-transform duration-200 ${openQuestion === index ? 'rotate-180' : ''}`}
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    stroke="currentColor"
+                  >
+                    <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M19 9l-7 7-7-7" />
+                  </svg>
+                </button>
+                <div 
+                  className={`px-5 py-3 bg-gray-700 bg-opacity-50 transition-all duration-200 ease-in-out ${
+                    openQuestion === index ? 'max-h-40 opacity-100' : 'max-h-0 opacity-0'
+                  }`}
+                >
+                  <p className="text-gray-300 text-xs leading-relaxed">{faq.answer}</p>
+                </div>
+              </div>
+            ))}
+          </div>
+        </div>
+      </section>
```

Comment:
You can create a new component name <FAQ /> move all faq logic there and then call the component [here](https://github.com/GlobeHoppin/GlobeHoppin/blob/main/src/pages/Homepage.jsx#L182).